File: C/program18.c
///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Display Hello on output screen 
// Autor       :  Ankita Anil Patil
// Date        :  05/05/2025
///////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

//Iteration

int Calculate (iNo)
{
    int iCnt = 0,iSum = 0;

    for(iCnt = 1; iCnt <= iNo; iCnt++ )
    {
        
        iSum = iSum + iCnt;
    }
    return iSum;
}

int main()
{
    int iValue = 0,iRet = 0;

    printf("Enter number\n" );
    scanf("%d",&iValue);

    iRet=Calculate(iValue);
    printf("Sum is : %d\n",iRet);

    return 0;
}

File: Java/program18.java
import java.util.*;

class DisplayClass
{
    public int Summation(int iNo)
    {
        int iCnt = 0, iSum =0;
        for(iCnt = iNo; iCnt >= 1; iCnt--)
        {
            iSum = iSum +  iCnt;                
        }

        return iSum;
    }
}

class program18
{
    public static void main(String A[])
    {
        int iValue = 0, iRet = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the frequency :");
        iValue = sobj.nextInt();

        DisplayClass dobj = new DisplayClass();
        iRet = dobj.Summation(iValue);


        System.out.println("Sum is :"+iRet);        

        
    }
}